src/Http/Http.Extensions/src/QueryBuilder.cs

```diff
@@ -23,6 +25,12 @@ public QueryBuilder(IEnumerable<KeyValuePair<string, string>> parameters)
             _params = new List<KeyValuePair<string, string>>(parameters);
         }
 
+        public QueryBuilder(IEnumerable<KeyValuePair<string, StringValues>> parameters)
+            : this(parameters.SelectMany(kvp => kvp.Value, (kvp, v) => KeyValuePair.Create(kvp.Key, v.ToString())))
```

Comment:
Looks inefficient. What's the scenario for this overload?

Reply:
To support the return by [`QueryHelpers.ParseQuery`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities.queryhelpers.parsequery?view=aspnetcore-3.1)